man/rmd-fragments/sidebar-configuration.Rmd

```diff
@@ -31,7 +31,7 @@ These are drawn from seven built-in components:
 
 -   `authors`: selected authors from the `DESCRIPTION`.
 
--   `dev`: development status badges found in `README.md`/`index.md`.
+-   `dev`: development status badges found in `README.md`/`index.md`. *This section is automatically removed for websites of released versions (see "Development mode" in `?build_site`. For all development modes, the badges are removed from the main content.*
```

Comment:
Rather than saying when it's removed, might be clearer to state when it's shown?

Reply:
Is the new phrasing any better? See also #2240: if we add a parameter, then the docs will need to change again :zany_face: